### ISSUE TITLE
Trivial typo in twisted.internet.protocol

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3182]
|<img alt="winjer's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| winjer reported|
|-|-|
|Trac ID|trac#3182|
|Type|defect|
|Created|2008-04-13 14:33:03Z|

Joule is spelled incorrectly as Jule, on line 218 of [twisted/internet/protocol.py](twisted/internet/protocol.py)

Attachments:

* [joule-3182.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/4c2/4c27a57dce667a0156a99cf4d327c6ddbfec7cb5/a8d3b604a73f2d463eb9c6411093b624f5c44247.patch) (563 bytes) - added by collab on 2008-07-01 23:57:14Z - patch against r24208

<details><summary>Searchable metadata</summary>

```
trac-id__3182 3182
type__defect defect
reporter__winjer winjer
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1208097183000000 1208097183000000
changetime__1215004390000000 1215004390000000
version__None None
owner__ 
cc__collab
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="collab's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|collab set owner to collab<br>collab set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

The value for the molar planck constant is also wrong, I think.  But maybe its within the error for the value.

**Comment by twisted-trac** [comment]
|<img alt="collab's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|collab removed owner<br>collab set status to `new`|
|-|-|

Yeah according to [this page](http://www.webqc.org/constants.php) the value for the 'molar Planck constant times c' value is 0.11962656572 instead of the current 0.11962656492 but yeah that uncertainty.. ;)

Attached [[joule-3182.patch](None/joule-3182.patch) patch] for both issues.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

0.11962656572 seems to be the 2002 CODATA value.  The 2006 CODATA value is 0.11962656472, so I think I'll use that (and add a reference to nist).  Thanks for finding a third value that I hadn't seen before, which gave me sufficient motivation to track down where the original value came from (1998 CODATA) and do enough investigation to be confident in the correctness of a new value. :)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [24214]) Apply joule-3182.patch, fixing a spelling error and refining the value of a physical constant

Author: collab
Reviewer: exarkun
Fixes: [#3182](https://github.com/twisted/twisted/issues/3182)

Fix the spelling of the word "joule" and provide a new value, taken
from 2006 CODATA recommended values (the previous value was from
1998 CODATA recommended values).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
